aw_datastore/storages/abstract.py

```diff
@@ -35,11 +35,11 @@ def get_metadata(self, bucket_id: str) -> dict:
 
     @abstractmethod
     def get_events(self, bucket_id: str, limit: int,
-                   starttime: Optional[datetime]=None, endtime: Optional[datetime]=None) -> List[Event]:
+                   starttime: datetime=None, endtime: datetime=None) -> List[Event]:
         raise NotImplementedError
 
     def get_eventcount(self, bucket_id: str,
-                   starttime: Optional[datetime]=None, endtime: Optional[datetime]=None) -> int:
+                   starttime: datetime=None, endtime: datetime=None) -> int:
```

Comment:
Why remove Optional?

Reply:
It's implicit when the default argument is None.

Reply:
Maybe for type hinting, but for actual type checking for mypy that doesn't make sense

Reply:
It seems that implicit `Optional` is frowned upon (but allowed, for now): https://github.com/python/typing/issues/275